Uppsala/UppQ1524o.xml

```diff
@@ -0,0 +1,627 @@
+                                <title type="complete" ref="LIT7073QeneTamayta"/>
+                                <incipit xml:lang="gez">
+                                    ተመይጠት፡ ጽዮን፡ ጼዋ፡ አጵሎን፡ ነኪር፤
+                                </incipit>
+                                <note>In the upper left corner of the folio, the following note has been written: 
+                                    "G.L."</note>
+                            </msItem>
+                            
+                            
+                            <msItem xml:id="p1_i20">
+                                <locus target="#40r #41r"/>
+                                <title type="complete" ref="LIT7072QeneMamhera"/>
+                                <incipit xml:lang="gez">
+                                    መምህረ፡ ዝንቱ፡ ዓለም፡ አውስጣቴዎስ፡ ደብረ፡ አርማንያ፡ ወልደከ፡ በጽሐ፡ ለዘምሮ፤
+                                </incipit>
+                                <note>In the upper left corner of the folio, the following note has been written: 
+                                    "G.L."</note>
+                            </msItem>
+                            
+                            
+                            <msItem xml:id="p1_i21">
+                                <locus target="#42r"/>
+                                <title type="complete" ref="LIT7099QeneEffoHalloka"/>
+                                <incipit xml:lang="gez">
+                                    <choice><sic>አ</sic><corr resp="JK">እ</corr></choice>ፎ፡ ሀሎከ፡ በጊዜ፡ 
+                                    ትቤ<del rend="overUnderlined strikethrough">ሳ</del><add place="above">ላ</add>፡ ኦሪተ፡ ሙሴ፡ ቅዱስ፤
+                                </incipit>
+                                <note>In the upper left corner of the folio, the following note has been written: 
+                                    "G.L."</note>
+                            </msItem>
+                            
+                            
+                            <msItem xml:id="p1_i22">
+                                <locus target="#45r #46r"/>
+                                <title type="complete" ref="LIT7101QeneAmh"/>
+                                <incipit xml:lang="am">
+                                    <choice><sic>ኢ</sic><corr resp="HSM">እየ</corr></choice>ደ<choice><sic>ርስበ</sic><corr resp="HSM">ረሰብ</corr></choice>ኝ፡ 
+                                    ነው፡ በሰኔ፡ ፈረሰኛው፡ ሞት፡ ነፍሴን፡ ሳልጠቅማት፡ ወየው፡ ሰዎች፡ 
+                                    <choice><sic>ኢ</sic><corr resp="HSM">እ</corr></choice>ያሉ፡ አዝማሪ፤
+                                </incipit>
+                                <note>In the upper left corner of the folio, the following note has been written: 
+                                    "G.L." In the upper right corner (under the <foreign xml:lang="gez">qəne</foreign> number),
+                                there is a note in Swedish: "<foreign xml:lang="sv">amhar</foreign>" ("Amharic"). Under this,
```

Comment:
Would you happen to have any idea about the hands here? You decided to encode it as a note, not as an addition. It may be correct, but then we still need to know the relationship between the main content and the notes.